doc/source/whatsnew/v1.3.2.rst

```diff
@@ -45,6 +45,7 @@ Bug fixes
 Other
 ~~~~~
 - :meth:`pandas.read_parquet` now supports reading nullable dtypes with ``fastparquet`` versions above 0.7.1.
+- Clarified suggested fix for ``PerformanceWarning`` emitted with a highly fragmented :class:`DataFrame` (:issue:`42579`)
```

Comment:
the release note above this one is not yet on 1.3.x (not backported), so this may need a manual backport.

Alternatively, I think we don't normally add a release note when just a message is changed.

Reply:
Ok sounds good, have just removed note then